src/target/riscv/riscv-013.c

```diff
@@ -2941,6 +2945,7 @@ static int execute_fence(struct target *target)
 			}
 			LOG_TARGET_DEBUG(target, "Unable to execute fence");
 		}
+		LOG_TARGET_DEBUG(target, "Successfully executed fence");
```

Comment:
```suggestion
		LOG_TARGET_DEBUG(target, "Successfully executed fence.i and fence rw, rw");
```

Reply:
Addressed, also modified another log in the previous two lines